### PR TITLE
bugfixes

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
0781959 | Removed unnecessary tags

### CODE REVIEW

File: reports/Tenant Overdue Alerts Activities.json [hunk 1]
```diff
@@ -2,20 +2,14 @@
...
-    "recordTags": [],
     "displayName": "Tenant Overdue Alerts Activities",
     "type": "rows",
     "templateType": "reporting",
     "roles": [
         "\/api\/3\/roles\/17562d60-b841-4ed7-912b-1ebaf063c8c9",
         "\/api\/3\/roles\/21d7a7c6-9d68-438f-b732-e8950c5f745b",
         "\/api\/3\/roles\/a501c11e-15af-49a6-a3a4-88605f32f016",
-        "\/api\/3\/roles\/c5ff8174-b3fa-41c5-b781-2b890b426a80",
-        "\/api\/3\/roles\/aada6692-fbf4-4a02-a580-f8a200962365"
```
Comment (cs-suyog-jinturkar) [inline reports/Tenant Overdue Alerts Activities.json:18]: which roles were removed and added?

File: reports/Tenant Overdue Incidents Activities.json [hunk 2]
```diff
@@ -2,20 +2,14 @@
...
-    "recordTags": [],
     "displayName": "Tenant Overdue Incidents Activities",
     "type": "rows",
     "templateType": "reporting",
     "roles": [
         "\/api\/3\/roles\/17562d60-b841-4ed7-912b-1ebaf063c8c9",
         "\/api\/3\/roles\/21d7a7c6-9d68-438f-b732-e8950c5f745b",
         "\/api\/3\/roles\/a501c11e-15af-49a6-a3a4-88605f32f016",
-        "\/api\/3\/roles\/c5ff8174-b3fa-41c5-b781-2b890b426a80",
-        "\/api\/3\/roles\/aada6692-fbf4-4a02-a580-f8a200962365"
```
Comment (cs-suyog-jinturkar) [inline reports/Tenant Overdue Incidents Activities.json:18]: which roles were removed and added?

File: reports/Tenant Weekly Alert Report.json [hunk 3]
```diff
@@ -2,20 +2,14 @@
...
-    "recordTags": [],
     "displayName": "Tenant Weekly Alert Report",
     "type": "rows",
     "templateType": "reporting",
     "roles": [
         "\/api\/3\/roles\/17562d60-b841-4ed7-912b-1ebaf063c8c9",
         "\/api\/3\/roles\/21d7a7c6-9d68-438f-b732-e8950c5f745b",
         "\/api\/3\/roles\/a501c11e-15af-49a6-a3a4-88605f32f016",
-        "\/api\/3\/roles\/c5ff8174-b3fa-41c5-b781-2b890b426a80",
-        "\/api\/3\/roles\/aada6692-fbf4-4a02-a580-f8a200962365"
```
Comment (cs-suyog-jinturkar) [inline reports/Tenant Weekly Alert Report.json:18]: which roles were removed and added?

File: reports/Tenant Weekly Incident Report.json [hunk 4]
```diff
@@ -2,20 +2,14 @@
...
-    "recordTags": [],
     "displayName": "Tenant Weekly Incident Report",
     "type": "rows",
     "templateType": "reporting",
     "roles": [
         "\/api\/3\/roles\/17562d60-b841-4ed7-912b-1ebaf063c8c9",
         "\/api\/3\/roles\/21d7a7c6-9d68-438f-b732-e8950c5f745b",
         "\/api\/3\/roles\/a501c11e-15af-49a6-a3a4-88605f32f016",
-        "\/api\/3\/roles\/c5ff8174-b3fa-41c5-b781-2b890b426a80",
-        "\/api\/3\/roles\/aada6692-fbf4-4a02-a580-f8a200962365"
```
Comment (cs-suyog-jinturkar) [inline reports/Tenant Weekly Incident Report.json:18]: which roles were removed and added?

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added review comments

File: reports/Tenant Overdue Alerts Activities.json (hunk 1 above)
Comment (cs-sujeet-ahirrao) [reply reports/Tenant Overdue Alerts Activities.json:18]: No new Role is added only the T2 analyst Role is converted to SOC Analyst and added that in Report.

File: reports/Tenant Overdue Incidents Activities.json (hunk 2 above)
Comment (cs-sujeet-ahirrao) [reply reports/Tenant Overdue Incidents Activities.json:18]: No new Role is added only the T2 analyst Role is converted to SOC Analyst and added that in Report.

File: reports/Tenant Weekly Alert Report.json (hunk 3 above)
Comment (cs-sujeet-ahirrao) [reply reports/Tenant Weekly Alert Report.json:18]: No new Role is added only the T2 analyst Role is converted to SOC Analyst and added that in Report.

File: reports/Tenant Weekly Incident Report.json (hunk 4 above)
Comment (cs-sujeet-ahirrao) [reply reports/Tenant Weekly Incident Report.json:18]: No new Role is added only the T2 analyst Role is converted to SOC Analyst and added that in Report.

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**